### ISSUE TITLE
Disappearing group indicator icons when units go uncon

### ISSUE THREAD

**Issue by igorpc1** [acemod/ACE3#8022]
**Mods (complete and add to the following information):**
- **Arma 3:** `2.00 stable`
- **CBA:** `3.15.2 stable`
- **ACE3:** `3.13.5 stable`

**Description:**
Group indicator icons disappear after someone goes into uncon and reappear after unit finally dies.

**Steps to reproduce:**

1. Go into editor
2. Spawn unit into your squad
3. Shoot him until he goes uncon
4. Watch icon disappering

**Expected behavior:**
Group indicator stays until unit reported dead like in previous version of ACE.

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer) / Virtual Arsenal

**Comment by BaerMitUmlaut** [comment]
I don't think we can influence those indicators. If you need those indicators, maybe try [DUI](https://steamcommunity.com/sharedfiles/filedetails/?id=1638341685).

**Comment by igorpc1** [comment]
Just funny why after medical rewrite they disappear, while in older version they work just fine. but okay then, thank you

**Comment by BaerMitUmlaut** [comment]
We changed the way we make AI not shoot at unconscious units which solved a bunch of other issues.
You mean the hexagons, right?

**Comment by igorpc1** [comment]
Yes, the hexagon. Very convenient to find your friends, because they fall uncon silently

**Comment by igorpc1** [comment]
So, i guess, by fixing AI we killed hexagon icons, right? I forgot to check, but what about map markers? Are they still persistent?

**Comment by 475fgdawger** [comment]
I am experiencing this issue with the recent ACE update
